health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/DrawerFieldComposer.js

```
@@ -276,220 +331,226 @@ function DrawerFieldComposer() {
+            }}
+            footerChildren={[
+              <Button
+                className={"app-config-pop-button"}
+                type={"button"}
+                size={"large"}
+                variation={"secondary"}
+                label={t("DIGIT_CLOSE")}
+                onClick={() => {
+                  setShowPopup(false);
+                }}
+              />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)